.github/ISSUE_TEMPLATE/other.yml

```
@@ -0,0 +1,9 @@
+name: Other
+description: Create a feature request, enhancement, proposal or suggestion
```

```suggestion
description: Create a feature request, enhancement, and proposal
```

I would move suggestions or things that are about seeking feedback and validation before creating issues.